.github/workflows/tests.yml

```diff
@@ -18,17 +18,8 @@ jobs:
     name: Tests
     strategy:
       matrix:
-        python-version: ['3.8', '3.9', '3.10']
+        python-version: ["3.9", "3.10"]
         database: [sqlite, postgres, mysql]
-        exclude:
```

Comment:
At the same time, excluding some test combinations makes testing faster and more likely to be actually performed in full. Is there a reason you removed these excludes?